src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/StrandOddsRatio.java

```diff
@@ -37,11 +37,11 @@
  *
  * <p>The sum R + 1/R is used to detect a difference in strand bias for REF and for ALT (the sum makes it symmetric). A high value is indicative of large difference where one entry is very small compared to the others. A scale factor of refRatio/altRatio where</p>
  *
- * $$ refRatio = \frac{max(X[0][0], X[0][1])}{min(X[0][0], X[0][1} $$
+ * $$ refRatio = \frac{min(X[0][0], X[0][1])}{max(X[0][0], X[0][1} $$
```

Comment:
- Missing a closing `]`
- If we are changing the equations here, then I believe we should also update AS_StrandOddsRatio.

The forum does not translate javadoc/gatkdoc LaTex nor ever did it in the past as far as I'm aware. The javadoc looks like this:

![screenshot 2019-03-06 14 48 16](https://user-images.githubusercontent.com/11543866/53909200-2a56b480-401f-11e9-90d0-c5a2d6db33f3.png)

--- 
If we take these to a LaTeX renderer, the equations look thusly:

![screenshot 2019-03-06 14 52 19](https://user-images.githubusercontent.com/11543866/53909325-7570c780-401f-11e9-8695-5ee72140a03a.png)

---

Are you okay with continuing to present the LaTeX equations @ldgauthier? Here are some choices to consider.

1. Adding images of the equations. This will be slightly different than the approach we took to updating [Article#11074](https://software.broadinstitute.org/gatk/documentation/article?id=11074). I believe it is a matter of html sourcing images within the javadoc portion of the code and uploading images to the forum. 
2. Looking into the forum CSS style sheets and figuring out how to dynamically render LaTeX equations. 
3. Keep the status quo but also provide a link to an online LaTeX renderer for folks to render.